DSharpPlus/Entities/Interaction/DiscordInteractionApplicationCommandCallbackData.cs

```diff
@@ -33,5 +34,5 @@ internal class DiscordInteractionApplicationCommandCallbackData
     public IReadOnlyCollection<DiscordAutoCompleteChoice> Choices { get; internal set; }
     
     [JsonProperty("poll", NullValueHandling = NullValueHandling.Ignore)]
-    public DiscordPoll Poll { get; internal set; }
+    public PollCreatePayload? Poll { get; internal set; }
```

Comment:
can we take a DiscordPoll here instead?

Reply:
The class `DiscordInteractionApplicationCommandCallbackData` is internal here, and used for building request payloads. This one cannot be changed in the style requested.